helfomat-core/src/main/java/de/helfenkannjeder/helfomat/core/organisation/OrganisationId.java

```diff
@@ -10,11 +12,13 @@
 /**
  * @author Valentin Zickner
  */
+@Embeddable
 public class OrganisationId implements Serializable {
 
     private static final Pattern VALID_UUID = Pattern.compile("^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$");
 
     @SuppressWarnings("CanBeFinal")
+    @Column(name = "organizationId")
     private String value;
```

Comment:
Why aren't you using UUID but instead validate a String?

Reply:
Not sure how that would behave during the elasticsearch indexing. But it would be unrelated to this change.